src/scene/container/Container.ts

```diff
@@ -377,7 +377,7 @@ export class Container<C extends ContainerChild = ContainerChild> extends EventE
     public didChange = false;
     // same as above, but for the renderable
     /** @private */
-    public didViewUpdate = false;
+    public didViewUpdate = true;
```

Comment:
Do we still need this to be a property of `Container`. Shouldn't this be only a property on `ViewContainer`?